src/states/default.rs

```diff
@@ -6,41 +6,44 @@ pub struct DefaultState;
 impl minigene::State for DefaultState {
     fn update(
         &mut self,
-        world: &mut World,
+        _world: &mut World,
```

Comment:
instead of changing those, just before the function name use
```
#[allow(unused_variables)]
```
(name might not be unused_variables, but you get the idea that it is the name of the warning you get when compiling)